cmd/root.go

```diff
@@ -38,3 +43,33 @@ func NewRootCmd() *cobra.Command {
 
 	return cmd
 }
+
+func runE(cmd *cobra.Command) error {
+	logLevelFlag, _ := cmd.Flags().GetString("loglevel")
+
+	if err := addLogger(logLevelFlag); err != nil {
+		return err
+	}
+	return nil
```

Comment:
```suggestion
	return addLogger(logLevelFlag); 
```
WDYT?

Reply:
I left this as it is, since here we could add different things like viper configuration, etc and this syntax could be useful. Please see https://github.com/reanahub/reana-client-go/pull/60/files#diff-ab967ab1a2f3a1b769106eeb7bfe892ef0e81d1d27811fa15be08e6749feee1fR49-R65